charts/karpenter-crd/templates/karpenter.azure.com_aksnodeclasses.yaml

```diff
@@ -249,6 +249,59 @@ spec:
                   - type
                   type: object
                 type: array
+              k8sVersion:
```

Comment:
IMO, avoid abbreviations in user-facing names.

`kubernetesVersion`, not `k8s`.

Reply:
Yea, I was wondering about the naming here. Makes sense to go with full name, rather than k8s. Will update.